docs/openapi.json

```diff
@@ -2,7 +2,7 @@
   "info": {
     "description": "Submit workflows to be run on REANA Cloud",
     "title": "REANA Server",
-    "version": "0.9.1"
```

Comment:
BTW a tiny independent change to MANIFEST.in that we could put into this release concerns:

```
include scripts/*
```

that could be better expressed as:

```
recursive-include scripts *.py
recursive-include scripts *.sh
```

For the consistency with other commands here and with what we do in other components.

See also
https://github.com/reanahub/reana-workflow-engine-snakemake/pull/72#discussion_r1412113033